data/json/mapgen/house/house32.json

```diff
@@ -1,10 +1,9 @@
 [
   {
+    "type": "mapgen",
     "method": "json",
     "om_terrain": "house_32",
     "//": "used by lab basements.",
-    "type": "mapgen",
-    "weight": 100,
```

Comment:
Does it not need a weight?

Reply:
As long as there's no other mapgen variant with the same id, it should be fine.